### ISSUE TITLE
Absent "When URL contains" field

### ISSUE THREAD

**Issue by kuzn123** [didierfred/SimpleModifyHeaders#54]
"When URL contains" field **missing** immediately after the first installation (or after reinstalling). It can appear only after importing the rules. v1.8.1.

**Comment by jwdevel** [comment]
Yeah, I just installed this extension for the first time (v1.8.1) and I was confused because there's no way to have a rule to only apply to certain URLs.

There is a `"url_contains":""` in the exported .conf file, but there is nothing in the UI to indicate it exists.
I have not found a way to make it appear, either.

Seems quite broken?

**Comment by jwdevel** [comment]
Ah, I found the "Filter URL per rules" setting; enabling that makes the column appear. A bit confusing, I must say.
